openapi/api.yml

```diff
@@ -6,20 +6,31 @@ info:
   version: 0.1.9
 
 paths:
-  /data:
+  /download:
     get:
-      operationId: core.controllers.examples.example
-      summary: Returns a list of data.
-      description: Optional extended description in CommonMark or HTML.
+      operationId: core.controllers.download.download
+      summary: Returns some data in a binary format.
+      description: Takes a file format and returns some data in that format.
```

Comment:
Guess it's only binary if excel, so maybe say, 'in requested format' or similar?

Reply:
changed this